data_loader/data_loader.py

```diff
@@ -0,0 +1,59 @@
+"""Database ingestion script."""
+import json
+import os
+
+import click
+import requests
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "setup_data/")
+STAC_API_BASE_URL = "http://localhost:8083"
+
+
+def load_data(filename):
+    """Load json data."""
+    with open(os.path.join(DATA_DIR, filename)) as file:
+        return json.load(file)
+
+
+def load_collection(collection_id):
+    """Load stac collection into the database."""
+    collection = load_data("collection.json")
+    collection["id"] = collection_id
+    try:
+        resp = requests.post(f"{STAC_API_BASE_URL}/collections", json=collection)
+        if resp.status_code == 200:
+            print(f"Status code: {resp.status_code}")
+            print(f"Added collection: {collection['id']}")
+        elif resp.status_code == 409:
+            print(f"Status code: {resp.status_code}")
+            print(f"Collection: {collection['id']} already exists")
+    except requests.ConnectionError:
+        click.secho("failed to connect")
+
+
+def load_items():
+    """Load stac items into the database."""
+    feature_collection = load_data("sentinel-s2-l2a-cogs_0_100.json")
+    collection = "test-collection"
+    load_collection(collection)
+
+    for feature in feature_collection["features"]:
```

Comment:
The Transactions Extension defines that either an Item or ItemCollection can be posted to /collections/{cid}/items -- if stac-fastapi doesn't support this, there should be a bug filed

Reply:
Good catch here!

Reply:
I just assumed that stac-fastapi was following the spec before

Reply:
Should we add this pr, then update the code it when the bulk transactions endpoint is fixed? Or should we wait?

Reply:
I'd wait -- this is fine for loading locally